src/Uno.UWP/UI/Input/Preview.Injection/InjectedInputRectangle.cs

```diff
@@ -11,4 +13,15 @@ public partial struct InjectedInputRectangle
 	public int Bottom;
 
 	public int Right;
+
+	public bool Equals(InjectedInputRectangle other) =>
+		Left == other.Left && Top == other.Top && Bottom == other.Bottom && Right == other.Right;
+
+	public override bool Equals(object? obj) => obj is InjectedInputRectangle other && Equals(other);
+
+	public override int GetHashCode() => new { Left, Top, Bottom, Right }.GetHashCode();
```

Comment:
```csharp
	public override int GetHashCode()
	{
		var hashCode = -1083629557;
		hashCode = hashCode * -1521134295 + Left.GetHashCode();
		hashCode = hashCode * -1521134295 + Top.GetHashCode();
		hashCode = hashCode * -1521134295 + Bottom.GetHashCode();
		hashCode = hashCode * -1521134295 + Right.GetHashCode();
		return hashCode;
	}
```

Reply:
Note: I got this from the lightbulb in VS (not sure if it will be suggested for you in Rider)

I usually just stick with whatever initial value suggested, but I think any large prime will do the trick.